### PR TITLE
More release notes

### PR DESCRIPTION
**Pull request by tmjd**
## Description
Trying to be a bit more prominent with the upgrade/migration requirements.

### CODE REVIEW

**Comment by tmjd** [comment]
image

File: _data/versions.yml [hunk 1]
```diff
@@ -8,10 +8,26 @@ v3.0:
     ##### Support for etcdv3
 
     - Calico now stores its data in [etcd version 3](https://coreos.com/blog/etcd3-a-new-etcd.html).
-    
+
+    > **Important**: If using the etcd datastore it is a requirement to use v3,
```
Comment (lxpollitt) [inline _data/versions.yml]: "requirement to use v3" probably doesn't give enough context or make it clear what v3 means.  e.g. Could be misunderstood as "you need to be running Calico v3 if you are using etcd", or perhaps "you need to be running etcd v3".

Perhaps something like this captures it better: "If you are upgrading a previous Calico deployment to Calico 3.0 you must follow this one-time migration process as part of the upgrade process.  Please read and make sure you fully understand the upgrade process before attempting to upgrade to Calico 3.0."

File: _data/versions.yml [hunk 2]
```diff
@@ -8,10 +8,26 @@ v3.0:
...
+    > and if upgrading a previous deployment it is necessary to migrate the
+    > data as part of the [upgrade process](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/)
+    {: .alert .alert-danger}
+
     ##### Migration and upgrade from v2.6.4
     
     - This version of Calico supports [migration and upgrade](/v3.0/getting-started/kubernetes/upgrade/) from Calico v2.6.4. 
+
+    - The Calico API has changed, which requires that Calico manifests, loaded using the `calicoctl` command, must be updated or converted to the new API.
```
Comment (lxpollitt) [inline _data/versions.yml]: Maybe "Calico resource model has been updated" would be clearer than "Calico API has changed"?  (People might not think of calicoctl as an API.)

Comment (tmjd) [reply _data/versions.yml]: I like "resource model", sounds good

**Comment by emanic** [comment]
Folded into https://github.com/projectcalico/calico/pull/1564

**Pull request closed without merging**